docs/articles/documentation/test-api/selecting-page-elements/selectors/selector-options.md

```diff
@@ -42,6 +42,59 @@ or the [selector-timeout](../../../using-testcafe/command-line-interface.md#--se
 
 `true` to additionally require the returned element to become visible within [options.timeout](#optionstimeout).
 
+This option is in effect when TestCafe waits for the selector to return a page element. This includes situations when
+
+* a property is obtained from the selector;
+
+    ```js
+    const width = await Selector('#element').with({ visibilityCheck: true }).clientWidth;
```

Comment:
In the begin of the topic we say: `You can pass the following options to the Selector constructor.`
I guess it would be better to see an example in this way to don't mix Selector options and the `.with` function

```js
const width = await Selector('#element', { visibilityCheck: true }).clientWidth;
```